modules/Core/pages/panel/general_settings.php

```diff
@@ -218,19 +218,6 @@
                 $errors = [$language->get('admin', 'config_not_writable')];
             }
 
-            /*
```

Comment:
We will have to remember to remove the `allowed_proxies` and `allowed_proxies_info` translations in #2658